### PR TITLE
docs: Updates to GeoMap panel doc

### PR DESCRIPTION
**Pull request by srperf**
**What is this feature?**

Multiple fixes and edits on GeoMap panel doc
- Outdated Data and Location Mode sections
- Updated images and descriptions for Markers layer settings
- Updated images and descriptions for Heatmap layer settings
- Added sections and descriptions for Day/Night, Photos, and Route.

**Why do we need this feature?**

Multiple elements in Geo Map docs were outdated

**Who is this feature for?**

Any Grafana user needing documentation on the Geo Map panel

**Which issue(s) does this PR fix?**:

None

**Special notes for your reviewer:**

Please check that:
- [x] It works as expected from a user's perspective.
- [ ] If this is a pre-GA feature, it is behind a feature toggle.
- [x] The docs are updated, and if this is a [notable improvement](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/#how-to-determine-if-content-belongs-in-a-whats-new-document), it's added to our [What's New](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/) doc.

### CODE REVIEW

File: docs/sources/panels-visualizations/visualizations/geomap/index.md [hunk 1]
```diff
@@ -270,8 +270,9 @@ The markers layer allows you to display data points as different marker shapes s
 
 ![Markers Layer](/static/img/docs/geomap-panel/geomap-markers-8-1-0.png)
 
-![Markers Layer Options](/static/img/docs/geomap-panel/geomap-markers-options-8-1-0.png)
+![Markers Layer Options](/media/docs/grafana/panels-visualizations/geomap-markers-options-11-1-0.png)
 
+- **Data and Location mode** configures the data setings for the layer. For more information check [Data](#data) and [Location mode](#location-mode).
```
Comment (imatwawana) [inline docs/sources/panels-visualizations/visualizations/geomap/index.md:275]:
```suggestion
- **Data** and **Location mode** configure the data settings for the layer. For more information, refer to [Data](#data) and [Location mode](#location-mode).
```

File: docs/sources/panels-visualizations/visualizations/geomap/index.md [hunk 2]
```diff
@@ -294,8 +295,9 @@ Similar to `Markers`, you are prompted with various options to determine which d
 
 ![Heatmap Layer](/static/img/docs/geomap-panel/geomap-heatmap-8-1-0.png)
 
-![Heatmap Layer Options](/static/img/docs/geomap-panel/geomap-heatmap-options-8-1-0.png)
+![Heatmap Layer Options](/media/docs/grafana/panels-visualizations/geomap-heatmap-options-11-1-0.png)
 
+- **Data and Location mode** configures the data setings for the layer. For more information check [Data](#data) and [Location mode](#location-mode).
```
Comment (imatwawana) [inline docs/sources/panels-visualizations/visualizations/geomap/index.md:300]:
```suggestion
- **Data** and **Location mode** configure the data settings for the layer. For more information, refer to [Data](#data) and [Location mode](#location-mode).
```

File: docs/sources/panels-visualizations/visualizations/geomap/index.md [hunk 3]
```diff
@@ -345,6 +347,7 @@ The Night / Day layer displays night and day regions based on the current time r
 
 ### Options
 
+- **Data** configures the data set for the layer. For more information check [Data](#data).
```
Comment (imatwawana) [inline docs/sources/panels-visualizations/visualizations/geomap/index.md:350]:
```suggestion
- **Data** configures the data set for the layer. For more information, refer to [Data](#data).
```

File: docs/sources/panels-visualizations/visualizations/geomap/index.md [hunk 4]
```diff
@@ -369,6 +372,7 @@ The Route layer renders data points as a route.
 
 ### Options
 
+- **Data and Location mode** configures the data setings for the layer. For more information check [Data](#data) and [Location mode](#location-mode).
```
Comment (imatwawana) [inline docs/sources/panels-visualizations/visualizations/geomap/index.md:375]:
```suggestion
- **Data** and **Location mode** configure the data settings for the layer. For more information, refer to [Data](#data) and [Location mode](#location-mode).
```

File: docs/sources/panels-visualizations/visualizations/geomap/index.md [hunk 5]
```diff
@@ -397,6 +401,7 @@ The Photos layer renders a photo at each data point.
...
```
Comment (imatwawana) [inline docs/sources/panels-visualizations/visualizations/geomap/index.md:404]:
```suggestion
- **Data** and **Location mode** configure the data settings for the layer. For more information, refer to [Data](#data) and [Location mode](#location-mode).
```

File: docs/sources/panels-visualizations/visualizations/geomap/index.md [hunk 6]
```diff
@@ -424,13 +429,14 @@ The Photos layer renders a photo at each data point.
...
 
-The Network layer renders a network graph. This layer supports the same [data format supported by the node graph visualization](ref:data-format-supported-by-the-node-graph-visualization) with the addition of [geospatial data]({{< relref "#location">}}) included in the nodes data. The geospatial data is used to locate and render the nodes on the map.
+The Network layer renders a network graph. This layer supports the same [data format supported by the node graph visualization](ref:data-format-supported-by-the-node-graph-visualization) with the addition of [geospatial data]({{< relref "#location-mode">}}) included in the nodes data. The geospatial data is used to locate and render the nodes on the map.
 
 {{< figure src="/media/docs/grafana/screenshot-grafana-10-1-geomap-network-layer-v2.png" max-width="750px" caption="Geomap network layer" >}}
 {{< video-embed src="/media/docs/grafana/screen-recording-10-1-geomap-network-layer-from-node-graph.mp4" max-width="750px" caption="Node graph to Geomap network layer" >}}
 
 ### Options
 
+- **Data and Location mode** configures the data setings for the layer. For more information check [Data](#data) and [Location mode](#location-mode).
```
Comment (imatwawana) [inline docs/sources/panels-visualizations/visualizations/geomap/index.md:439]:
```suggestion
- **Data** and **Location mode** configure the data settings for the layer. For more information, refer to [Data](#data) and [Location mode](#location-mode).
```

File: docs/sources/panels-visualizations/visualizations/geomap/index.md [hunk 7]
```diff
@@ -424,13 +429,14 @@ The Photos layer renders a photo at each data point.
 The Network layer is currently in [public preview](/docs/release-life-cycle/). Grafana Labs offers limited support, and breaking changes might occur prior to the feature being made generally available.
 {{% /admonition %}}
 
-The Network layer renders a network graph. This layer supports the same [data format supported by the node graph visualization](ref:data-format-supported-by-the-node-graph-visualization) with the addition of [geospatial data]({{< relref "#location">}}) included in the nodes data. The geospatial data is used to locate and render the nodes on the map.
+The Network layer renders a network graph. This layer supports the same [data format supported by the node graph visualization](ref:data-format-supported-by-the-node-graph-visualization) with the addition of [geospatial data]({{< relref "#location-mode">}}) included in the nodes data. The geospatial data is used to locate and render the nodes on the map.
```
Comment (imatwawana) [inline docs/sources/panels-visualizations/visualizations/geomap/index.md:432]: What's the page that you're linking to in the first link?

File: docs/sources/panels-visualizations/visualizations/geomap/index.md [hunk 8]
```diff
@@ -294,8 +295,9 @@ Similar to `Markers`, you are prompted with various options to determine which d
 
 ![Heatmap Layer](/static/img/docs/geomap-panel/geomap-heatmap-8-1-0.png)
 
-![Heatmap Layer Options](/static/img/docs/geomap-panel/geomap-heatmap-options-8-1-0.png)
+![Heatmap Layer Options](/media/docs/grafana/panels-visualizations/geomap-heatmap-options-11-1-0.png)
```
Comment (imatwawana) [inline docs/sources/panels-visualizations/visualizations/geomap/index.md:298]: The image is quite large for the page
```suggestion
{{< figure src="/media/docs/grafana/panels-visualizations/geomap-heatmap-options-11-1-0.png" max-width="350px" alt="Heatmap layer options" >}}
```

File: docs/sources/panels-visualizations/visualizations/geomap/index.md [hunk 9]
```diff
@@ -270,8 +270,9 @@ The markers layer allows you to display data points as different marker shapes s
 
 ![Markers Layer](/static/img/docs/geomap-panel/geomap-markers-8-1-0.png)
 
-![Markers Layer Options](/static/img/docs/geomap-panel/geomap-markers-options-8-1-0.png)
+![Markers Layer Options](/media/docs/grafana/panels-visualizations/geomap-markers-options-11-1-0.png)
```
Comment (imatwawana) [inline docs/sources/panels-visualizations/visualizations/geomap/index.md:273]: The image is quite large when I look at it in preview so I've swapped out a regular markdown image line with a figure shortcode so we can control the size.
```suggestion
{{< figure src="/media/docs/grafana/panels-visualizations/geomap-markers-options-11-1-0.png" max-width="350px" alt="Markers layer options" >}}
```

**Review by imatwawana — COMMENTED** [review]
This is good! I'm not 100% sure we need the added images. Do you think folks are getting confused without them? I ask because adding images = maintaining images. So I always want to be intentional about adding them.

File: docs/sources/panels-visualizations/visualizations/geomap/index.md (hunk 7 above)
Comment (srperf) [reply docs/sources/panels-visualizations/visualizations/geomap/index.md:432]: Yes I created that one last week :)

Comment (imatwawana) [reply docs/sources/panels-visualizations/visualizations/geomap/index.md:432]: Is this the page and heading you're referring to? I have to add the link in the front matter of the page for the reference to work:
https://grafana.com/docs/grafana/latest/panels-visualizations/visualizations/node-graph/#data-requirements

Comment (imatwawana) [reply docs/sources/panels-visualizations/visualizations/geomap/index.md:432]: Oh sorry, I didn't see it when I looked in the front matter.

**Comment by srperf** [comment]
I defer to your opinion @imatwawana 
I don't have any strong opinions on having that image, so if it just adds complexity, lets take it out :)

**Review by imatwawana — APPROVED** [review]

**Pull request merged**